simulator/src/test/java/com/hedera/block/simulator/BlockStreamSimulatorTest.java

```diff
@@ -19,27 +19,37 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 import com.hedera.block.simulator.generator.BlockStreamManager;
+import com.hedera.block.simulator.grpc.PublishStreamGrpcClient;
 import com.swirlds.config.api.Configuration;
+import java.io.IOException;
+import java.util.Map;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
 @ExtendWith(MockitoExtension.class)
 class BlockStreamSimulatorAppTest {
```

Comment:
I think this class name needs to be `BlockStreamSimulatorTest`. In IntelliJ I think it's showing up as an inner class

Reply:
Good catch.

probably a refactor gonne wrong that I missed Thank you! 👍 

✅  Done!